x-pack/plugins/maps/public/connected_components/mb_map/transform_request.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import {
+  FONTS_API_PATH,
+  MVT_GETTILE_API_PATH,
+  MVT_GETGRIDTILE_API_PATH,
+} from '../../../common/constants';
+import { getHttp } from '../../kibana_services';
+
+const FONTS = getHttp().basePath.prepend(FONTS_API_PATH);
+const GETTILE = getHttp().basePath.prepend(MVT_GETTILE_API_PATH);
+const GETGRIDTILE = getHttp().basePath.prepend(MVT_GETGRIDTILE_API_PATH);
+
+export function transformRequest(url: string, resourceType: string | undefined) {
```

Comment:
💯  version handling when maplibre-gl handles the fetching was one of my initial concerns. you handled it perfectly here. 👍